typeclasses/rooms.py

```diff
@@ -7,7 +7,6 @@
 
 from evennia import DefaultRoom
```

Comment:
Room should be pulling from typeclasses.object.Object, and then you don't need the property code anymore. All 3 of these could be consolidated with a proper use of super. DefaultCharacter inherits from DefaultObject and just does some extra stuff to it. Object should have the properties, and Character and Room can both pull from Object. Character will need its locks.